src/vm/ceeload.cpp

```diff
@@ -3420,13 +3420,7 @@ BOOL Module::IsInCurrentVersionBubble()
 //
 BOOL Module::IsInSameVersionBubble(Module *target)
 {
-    CONTRACT(BOOL)
-    {
-        THROWS;
-        GC_TRIGGERS;
-        MODE_ANY;
-    }
-    CONTRACT_END;
+    STANDARD_VM_CONTRACT;
```

Comment:
I believe the below `RETURN`s will need to be regular `return` statements now since `STANDARD_VM_CONTRACT` doesn't have post-conditions.

Reply:
Right, I've missed that.